src/channel.cr

```diff
@@ -41,10 +57,18 @@ abstract class Channel(T)
     @closed
   end
 
+  # Receive a value from the channel.
```

Comment:
```suggestion
  # Receives a value from the channel.
  # If there is a value waiting, it is returned immediately. Otherwise, this method blocks until a value is sent to the channel.
  #
  # Raises `ClosedError` if the channel is closed or closes while waiting for receive.
```